test/integration/output_test.py

```diff
@@ -32,6 +32,7 @@ def fast_out(pudl_engine, pudl_datastore_fixture):
         fill_fuel_cost=FILL_FUEL_COST,
         roll_fuel_cost=True,
         fill_net_gen=False,
+        backfill_tech_desc=True
```

Comment:
Maybe make this `fill_tech_desc` if that's what we're using on all the others and in the documentation language (and it's not just backfilling)

Reply:
sounds logical to me